packages/fields/src/types/Content/serialiser.js

```diff
@@ -65,6 +65,14 @@ export function serialiseSlateValue(value, blocks) {
           );
         }
 
+        if (!block.path) {
+          throw new Error(
+            `No mutation path set for block view type '${
+              block.type
+            }'. Ensure the block's view exports a 'path' key corresponding to the mutation path for saving block data`
+          );
+        }
```

Comment:
There's a check for `.type`, but the other required `.path` didn't have a check so I added it.

Note: This `block` could be a _client-side_ block, so would not trigger the getter which throws an error if `get path()` isn't implemented (see `Block.js`), hence we need the check here.